samples/boards/nrf52/mesh/onoff_level_lighting_vnd_app/src/mesh/transition.c

```diff
@@ -14,8 +14,46 @@
 #include "state_binding.h"
 #include "transition.h"
 
-u8_t enable_transition;
-u8_t default_tt;
+static bool is_new_transition_start;
```

Comment:
Isn't the transition bound to a specific state, i.e. instead of being global shouldn't this variable be part of the state struct(s)?

Reply:
Currently, only one transition is allowed at a time. Using global variable here reduces complexity.
Here I wanna wait & see what PTS is expecting.

Reply:
Ok ... I will put "is_new_transition_start" in each relevant struct.

Reply:
Done !!

Reply:
I don't see this change. The start timestamp is still external to any structs.

Reply:
"is_new_transition_start" I was assuming about this variable. I've created separate entry in Stuct define for Servers

But now not getting what your are expecting ? Could you elaborate more on it or directly mentioned about that variable ?

Reply:
Done !!